mne/viz/evoked.py

```diff
@@ -3217,5 +3219,7 @@ def click_func(
     if cmap is not None:
         _draw_colorbar_pce(ax, _colors, _cmap, colorbar_title, colorbar_ticks)
     # finish
+    if topo_disp_title:
+        plt.suptitle(topo_title)
```

Comment:
Prefer methods to the global pipeline approach
```suggestion
        ax.figure.suptitle(topo_title)
```